upgrading/extending-daml-models/2_new_package.md

```diff
@@ -0,0 +1,91 @@
+In this step, you'll create a new package depending on `create-daml-app` from scratch. We will be
+implementing the back-end for a discussion forum. The users will be identified by their user data
+from the mini social network of the Getting Started Guide. Thus we don't need to reimplement the
+user roles.
+
+In the terminal, change directory to your home directory and create a new empty DAML project:
+
+```
+cd
+daml new forum
+cd forum
+```{{execute T1}}
+
+The first thing you need to do is to add the `create-daml-app` to the dependencies in the
+`daml.yaml`{{open}} file. After, the file will look like this:
```

Comment:
When you create a new project you also need to add it to the editor as it's not visible there. Not sure what's the best way to do it 🤷‍♂️I've done it like [this](https://github.com/digital-asset/daml-katacoda/blob/master/getting-started/your-first-feature/4_create_message_list.sh) for the "Your first feature" scenario

Reply:
I think the problem is that the working directory of the IDE is `create-daml-app`. So I need to set the path to `../forum/daml.yaml` and it works.